CovertInfiltration/Src/CovertInfiltration/Classes/UISkyrangerExfiltrate.uc

```diff
@@ -126,7 +121,7 @@ simulated function BuildScreen()
 
 	`XSTRATEGYSOUNDMGR.PlaySoundEvent("Geoscape_SkyrangerStop");
 	
-	NewGameState = class'XComGameStateContext_ChangeContainer'.static.CreateChangeState("Skyranger Aborts Event");
+	NewGameState = class'XComGameStateContext_ChangeContainer'.static.CreateChangeState("CI: Building SkyrangerExfiltrate Screen");
 	`XEVENTMGR.TriggerEvent('OnSkyrangerArrives', , , NewGameState);
```

Comment:
You changed the game state description - that is only visible in `X2DebugHistory` command.

What you needed to change is the event name:

```uc
`XEVENTMGR.TriggerEvent('OnSkyrangerArrives', , , NewGameState);
```

to something like 

```uc
`XEVENTMGR.TriggerEvent('OnExfiltrationSkyrangerArrives', , , NewGameState);
```